apps/component-generator/package-lock.json

```diff
@@ -0,0 +1,3086 @@
+{
+  "name": "component-generator",
```

Comment:
I don't think we should have package-lock.json files, we use yarn so we should only be using yarn.lock. Do you know where this came from? I also don't see any updates to the yarn.lock file for the new dependencies..

Reply:
I think it came from `npm i` command, I did this by mistake